### PR TITLE
removing memberships on the library header

### PR DESCRIPTION
**Pull request by ahsu1230**
@andrewconner @2is10 

Thanks @camhashemi for the suggestions

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/deleteMemberConfirmModal.tpl.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div kf-modal class="kf-remove-member-confirm-modal">
+  <div kf-basic-modal-content with-cancel="true" title="Are you sure?" with-warning action-text="Remove Member" action="removeMember()">
+    <p>Removing collaborators will prevent them from being able to add, edit or remove keeps in this library</p>
+  </div>
+</div>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/libraries/deleteMemberConfirmModal.tpl.html]: This file appears to be unused.

**Comment by ahsu1230** [comment]
@2is10 i was trying out the dropdown menu (to both remove or demote a collaborator to a follower), and it seemed really super weird when it was on the library header. I think the best UI is honestly to go with a small `x` on top of a user's image and then a confirmation modal once clicked.

**Comment by 2is10** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/libraries/deleteMemberConfirmModal.tpl.html (hunk 1 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/libraries/deleteMemberConfirmModal.tpl.html]: woops you're right. Missed that =/

**Pull request merged**